apps/docs/content/3-components/2-library/summary-list.mdx

```diff
@@ -7,24 +7,131 @@ libraries:
+    <SummaryListRow label="Date of Birth">
+      <SummaryListValue>8 November 1982</SummaryListValue>
+      <SummaryListAction href="/change-dob">
+        Change date of birth
+      </SummaryListAction>
+    </SummaryListRow>
+    <SummaryListRow label="Address">
+      <SummaryListValue>
+        {['72 Guild Street', 'London', 'SE23 6FH']}
+      </SummaryListValue>
+      <SummaryListAction href="/change-address">
+        Change address
+      </SummaryListAction>
+    </SummaryListRow>
+  </SummaryList>
+</ComponentContainer>
+
+<Tabs id="summary-list-default">
+  <TabList>
+    <TabItem value="html" checked>HTML</TabItem>
+    <TabItem value="macro">Macro</TabItem>
+    <TabItem value="react">React</TabItem>
+  </TabList>
+  <TabPanel value="html">
+  ```html
+  <div class="gi-summary-list" role="list" aria-label="Summary list">
+    <dl>
+      <dt>Name</dt>
+      <dd class="gi-summary-list-value">John Smith</dd>
+      <dd class="gi-summary-list-actions">
+        <a href="/change-name" aria-label="Link action: Change name">Change name</a>
+      </dd>
+    </dl>
+    <dl>
+      <dt>Date of Birth</dt>
+      <dd class="gi-summary-list-value">8 November 1982</dd>
+      <dd class="gi-summary-list-actions">
+        <a href="/change-dob" aria-label="Link action: Change date of birth">Change date of birth</a>
+      </dd>
+    </dl>
+    <dl>
+      <dt>Address</dt>
+      <dd class="gi-summary-list-value">
+        <p>72 Guild Street</p>
+        <p>London</p>
+        <p>SE23 6FH</p>
+      </dd>
+      <dd class="gi-summary-list-actions">
+        <a href="/change-address" aria-label="Link action: Change address">Change address</a>
+      </dd>
+    </dl>
+  </div>
+  ```
+  </TabPanel>
+  <TabPanel value="macro"> 
+    ```html 
+        {{ govieSummaryList({ "rows":
+          [ { "label": "Name", "value": "John Smith", "action": { "href": "/change-name", "label": "Change name" } },
+            { "label": "Date of Birth", "value": "8 November 1982", "action": { "href": "/change-dob", "label": "Change date of birth" } },
+            { "label": "Address", "value": ["72 Guild Street", "London", "SE23 6FH"], "action": { "href": "/change-address", "label": "Change address" } } 
+          ]})
+        }} 
+    ```
+  </TabPanel>
+  <TabPanel value="react">
+    ```react
+        import { SummaryList, SummaryListRow, SummaryListValue, SummaryListAction } from '@govie-ds/react';
+        
+        <SummaryList>
+          <SummaryListRow label="Name">
+            <SummaryListValue>John Smith</SummaryListValue>
+            <SummaryListAction href="/change-name">Change name</SummaryListAction>
+          </SummaryListRow>
+          <SummaryListRow label="Date of Birth">
+              <SummaryListValue>8 November 1982</SummaryListValue>
+              <SummaryListAction href="/change-dob">Change date of birth</SummaryListAction>
+          </SummaryListRow>
+          <SummaryListRow label="Address">
+            <SummaryListValue>
```

Comment:
@BogdanDanilescu @pregno 

Do you guys have any alternative ways of passing an array or object to the summary list value? I haven't seen this pattern tbh to pass JS as children 🤔

Reply:
Maybe this can be more convenient for the developer.

```
<SummaryListValue values={['72 Guild Street', 'London', 'SE23 6FH']} />
```

Reply:
good call, I'm good either way tbh, I was just trying to keep ```children``` prop following the pattern from the others summary list children components.
Let me know your toughts!

Reply:
Shouldn't the `<SummaryListValue />` accept strings ? Is there a reason why we are passing an array of strings here ?

Reply:
or alternatively, if we could do something like this:
```
<SummaryListValue>
  <Paragraph>72 Guild Street</Paragraph>
  <Paragraph>London</Paragraph>
  <Paragraph>SE23 6FH</Paragraph>
</SummaryListValue />
```

Reply:
I'm following the Local version, so there they are breaking lines ```<br/>``` so I ended up to having an array of strings.
So intead of adding a Paragraph that will include non desired styles, what about to have something like this:

```
<SummaryListValue>
   72 Guild Street
   <br/>
   London
   <br/>
   SE23 6FH
 <SummaryListValue>
 ```
 
 @BogdanDanilescu

Reply:
Yes, this works too. But just thinking about what the other devs were saying about giving them the ability to compose / add what they need depending on their use case. Having that in mind, would it be a good idea to have a `<Slot />` in the ``<SummaryListValue>`` and so it will be up to the developer what they want to render as a value. They can either do something similar to your example with the `<br />` or they can just do a mapping of paragraphs, whatever they think it will work best in their situation

Reply:
agree, this will bring more flexibility.